client/src/includes/sidePanel.js

```diff
@@ -155,10 +156,7 @@ export default function initSidePanel() {
       newWidth,
     ).replace('%(num)s', newWidth);
 
-    sidePanelWrapper.parentElement.style.setProperty(
-      '--side-panel-width',
-      `${newWidth}px`,
-    );
+    content.style.setProperty('--side-panel-width', `${newWidth}px`);
```

Comment:
```suggestion
    document.documentElement.style.setProperty(
      '--side-panel-width',
      `${newWidth}px`,
    );
```

This works in Chrome but needs to be checked across various browsers.

Essentially this sets a `:root` css custom property.

https://css-tricks.com/updating-a-css-variable-with-javascript/